src/services/asset-inventory/cloud-service/CloudServicePage.vue

```diff
@@ -173,24 +173,23 @@ export default {
         const searchQueryHelper = new QueryHelper();
         const state = reactive({
             hasManagePermission: useManagePermissionState(),
-            // asset inventory store
-            selectedProvider: computed(() => assetInventoryStore.state.cloudService.selectedProvider),
-            period: computed(() => assetInventoryStore.state.cloudService.period),
-            searchFilters: computed<ConsoleFilter[]>(() => assetInventoryStore.state.cloudService.searchFilters),
-            selectedCategories: computed<CloudServiceCategory[]>(() => assetInventoryStore.getters['cloudService/selectedCategories']),
-            selectedRegions: computed<string[]>(() => assetInventoryStore.getters['cloudService/selectedRegions']),
-            allFilters: computed<ConsoleFilter[]>(() => assetInventoryStore.getters['cloudService/allFilters']),
+            title: computed<string>(() => {
+                if (!isEmpty(storeState.providers[cloudServicePageState.selectedProvider])) {
+                    return storeState.providers[cloudServicePageState.selectedProvider].name;
+                }
+                return cloudServicePageState.selectedProvider;
+            }),
```

Comment:
question: I wonder why was this added 😮

Reply:
Original code was this (in template!)
I thought it was too long and unreadable, so change it to computed variable.

```
:title="storeState.providers[selectedProvider] ? storeState.providers[selectedProvider].name : selectedProvider"
```

Reply:
Oh, thanx for explanation 😄